worker/jobTypes/fastlyJob.js

```diff
@@ -0,0 +1,55 @@
+const mongo = require('../utils/mongo');
+
+class FastlyJobClass {
+    // pass in a job payload to setup class
+    constructor(currentJob) {
+        this.currentJob = currentJob;
+    }
+
+    // connects to dochub database and upsert {source: target} mappings
+    // to the fastly edge dictionary
+    connectAndUpsert(MongoClient, fastly, database, collection) {
+        mongo.initMongoClient();
+        var cursor = 
+        MongoClient.connect(mongo.url, function(err, client) {
+            assert.equal(null, err);
+        
+            const db = client.db(database);
+        
+            var cursor = db.collection(collection).find({});
+        
+            function iterateFunc(doc) {
+              const page = "https://dochub.mongodb.org/core/" + doc.name;
+              var request = require('request');
```

Comment:
We want to check to make sure that the target page is not 404ing. Before the dochub redirect is set up it likely will 404!

Reply:
Oh whoops! I think I accidentally deleted that code 👀